src/common/access_pattern.c

```diff
@@ -350,11 +352,12 @@ xdd_save_seek_list(target_data_t *tdp) {
 		}
 		average = (uint64_t)(total / sp->seek_total_ops);
 		/* Print the seek list into the specified file */
-		fprintf(tmp,"# Longest seek=%llu, Shortest seek=%llu, Average seek distance=%llu\n",
+		fprintf(tmp,"# Longest seek=%llu, Shortest seek=%llu, Average seek distance=%llu, Number of Request=%ld\n",
```

Comment:
`Number of Request` -> `Number of Requests`. Sorry, I forgot to put the `s` at the end when I last suggested changes.